tests/e2e/tests/DisasterType/SwitchBetweenDisasterType.spec.ts

```diff
@@ -33,7 +33,7 @@ test.beforeEach(async ({ page }) => {
   );
 });
 
-test(
+test.skip(
```

Comment:
please add a reason for skipping and an item for fixing it.

Reply:
Done!